### ISSUE TITLE
Remove alt text for logo throughout participant portal

### ISSUE THREAD

**Issue by ignatiusm** [Garvan-Data-Science-Platform/ctrl#720]
I've noticed a couple of places in the participant portal where alt-text is displayed (in black font) if a logo has not been set. This has been addressed in other places by removing alt-text. I'm mindful this negatively impacts accessibility, so maybe we need different logic to show alt-text, but not if a logo has not been set.

E.g. when reviewing a step within a consent survey:

<img width="205" height="112" alt="Image" src="https://github.com/user-attachments/assets/8e67c2a5-8abb-49bf-98f6-cba1b63a4caf" />
